python/ray/actor.py

```diff
@@ -1777,19 +1777,10 @@ def exit_actor():
             worker is not an actor.
     """
     worker = ray._private.worker.global_worker
-    if worker.mode == ray.WORKER_MODE and not worker.actor_id.is_nil():
-        # In asyncio actor mode, we can't raise SystemExit because it will just
-        # quit the asycnio event loop thread, not the main thread. Instead, we
-        # raise a custom error to the main thread to tell it to exit.
-        if worker.core_worker.current_actor_is_asyncio():
-            raise AsyncioActorExit()
-
-        # Set a flag to indicate this is an intentional actor exit. This
-        # reduces log verbosity.
-        raise_sys_exit_with_custom_error_message("exit_actor() is called.")
```

Comment:
We can probably leave this part in, else it's a behavior change (code after `exit_actor()` will now be called)